packages/vue-app/src/index.js

```diff
@@ -5,11 +5,13 @@ export const template = {
   dependencies,
   dir: path.join(__dirname, '..', 'template'),
   files: [
+    'nuxt/config.json',
     'App.js',
     'client.js',
     'index.js',
     'router.js',
     'router.scrollBehavior.js',
+    'router/routes.json',
```

Comment:
While I agree it is a cleaner structure, but we already have router.js and router.scrollBehaviour.js. We end up a directory with one file :D

Reply:
It could be moved to `routes.json` indeed